dlib/svm/function_abstract.h

```diff
@@ -282,15 +266,7 @@ namespace dlib
         !*/
 
         probabilistic_decision_function (
-            const probabilistic_decision_function& f
-        );
-        /*!
-            ensures
-                - #*this is a copy of f
-        !*/
-
-        probabilistic_decision_function (
-            const probabilistic_function<decision_function<K> >& d
+            const probabilistic_function<decision_function<K> >& f
```

Comment:
The comment speaks about `f`, so thought that at least one of these should be changed.